frontend/src/metabase/admin/settings/components/CloudPanel/MigrationError.tsx

```diff
@@ -0,0 +1,43 @@
+import { c, t } from "ttag";
+
+import Link from "metabase/core/components/Link";
+import { color } from "metabase/lib/colors";
+import { Flex, Icon, Text, Box } from "metabase/ui";
+import type { CloudMigration } from "metabase-types/api/cloud-migration";
+
+import { LargeIconContainer, MigrationCard } from "./CloudPanel.styled";
+import { getMigrationEventTime } from "./utils";
```

Comment:
nit: if we used `<DateTime value={timestamp} />` in this util it would format the date in a way that respects `MetabaseSettings.formattingOptions`